express-delete/server.js

```diff
@@ -0,0 +1,34 @@
+import express from 'express';
+
+const app = express();
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res, next) => {
+  res.json(Object.values(grades));
+  return next();
+});
```

Comment:
Since this is all configuring Express, I think it would be nice to put a blank line between each of the configuration settings. It makes them stand out better.